tests/simulator/test_initialization.py

```diff
@@ -603,21 +602,19 @@ def _test_density_decreases_as_1overSqrtN(self, dim, interp_order):
         plt.xlabel("nbr_particles")
         plt.legend()
         plt.savefig(
-            "noise_nppc_minus_theory_interp_{}_{}.png".format(dim, interp_order)
+            f"noise_nppc_minus_theory_interp_{dim}_{interp_order}.png"
```

Comment:
is this easy to get these images as artifacts in TC?

Reply:
should be, so far we only export images for functional tests in nightly builds, but we can change that